src/components/countdown/code/lib/countdown.js

```diff
@@ -0,0 +1,56 @@
+var moment = require('moment');
+var Countdown = {};
+module.exports = Countdown;
+
+var pad = function (number) {
+  number = Math.abs(Math.floor(number));
+  if (number > 9) {
+    return number;
```

Comment:
This should be returning a string so that the output of this function is a consistent type. i.e ```return '' + number;```